tests/build.gradle

```diff
@@ -49,11 +48,7 @@ dependencies {
     testCompile 'io.spray:spray-json_2.11:1.3.2'
     testCompile 'com.google.code.tempus-fugit:tempus-fugit:1.2-SNAPSHOT'
     testCompile 'com.google.code.gson:gson:2.3.1'
-
-    testCompile files('$whiskhome/common/scala/build/libs/scala.jar',
-                      '$whiskhome/core/controller/build/libs/controller.jar', 
-                      '$whiskhome/core/dispatcher/build/libs/dispatcher.jar')
-    testCompile files(test_common)
+    testCompile project(':WhiskTest').sourceSets.test.output
```

Comment:
@markusthoemmes @daisy-ycguo 
I think we have to have :common:scala, :core:dispatcher and :core:controller present in the projects. I will try a fresh start from scratch and check.